src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -2429,4 +2442,345 @@ public Response getTimestamps(@PathParam("identifier") String id) {
             return wr.getResponse();
         }
     }
+
```

Comment:
It would be helpful for reviewers to make it clear that there aren't any changes except formatting up to this point. Some of the comparisons are fairly confusing they way they are broken up.